src/components/Exercise/ExerciseDetails.js

```diff
@@ -30,31 +45,22 @@ const Heading = styled.div`
   font-size: 1.1rem;
 `
 const Title = styled.textarea`
-  margin-top: 15px;
   border: none;
-  padding: 0;
+  padding: ${props => (props.exerciseTitle ? 0 : 10)}px;
   outline: none;
   width: 100%;
-  height: 40px;
+  height: ${props => (props.exerciseTitle ? 40 : 400)}px;
   resize: none;
-  font-size: 1.7rem;
-  font-weight: bold;
-  background-color: #fff;
-  color: #333;
-`
-
-const Text = styled.textarea`
-  border: none;
-  outline: none;
-  width: 100%;
-  height: 400px;
-  resize: none;
-  font-size: inherit;
+  font-size: ${props => (props.exerciseTitle ? 1.7 : 1)}rem;
+  font-weight: ${props => (props.exerciseTitle ? 'bold' : 400)};
   background-color: #fff;
   color: #333;
-  padding: 10px;
 `
 const Image = styled.img`
   width: 25px;
   height: 25px;
 `
+const ButtonWrapper = styled.div`
```

Comment:
Würde es nicht ButtonWrapper sondern eher ControllsWrapper nennen, da du ja semantisch gesehen in dem Wrapper neben Buttons auch Links als Kind Elementen drin hast.

Da du in anderen Components auch eine Styled Component ButtonWrapper hast, wäre es eine Überlegung wert, diese evtl in eine eigene Datei auszulagern.